db/00000000000003_outstanding_batch_state.down.sql

```diff
@@ -0,0 +1,4 @@
+CREATE INDEX outstanding_batches_task_and_time_bucket_index ON outstanding_batches(task_id, time_bucket_start);
+DROP INDEX outstanding_batches_task_id_and_time_bucket_start CASCADE;
+ALTER TABLE outstanding_batches DROP COLUMN state;
+DROP TYPE OUTSTANDING_BATCH_STATE CASCADE;
```

Comment:
Similar arguments could apply with the down migration. Although I don't care that much about cascading deletes on a down migration.